recipes/mapio/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.7.2" %}
+
+package:
+  name: mapio
+  version: {{ version }}
+
+source:
+  url: https://github.com/usgs/MapIO/archive/{{ version }}.tar.gz
+  sha256: 711e5fd5170c17cdf8aba71eedd9c6c7c6f40d5a683a974612f96189cde48013
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=3
```

Comment:
@mhearne-usgs with this "trick" we can avoid the `skip: True  # [py2k]` and make recipes like this one `noarch`.